### ISSUE TITLE
Incorrect lint errors about single quotes

### ISSUE THREAD

**Issue by petey** [AtomLinter/linter-docker#31]
I'm not sure if anyone is still maintaining this plugin, but:
For linter-docker 0.1.4 with linter 1.11.21

I get the error `Arguments to ENTRYPOINT in exec must not contain single quotes at line 2 col 1` for the following Dockerfile

```
FROM alpine:latest
ENTRYPOINT [ "/bin/bash" ]
```

This Dockerfile doesn't contain single quotes in it anywhere.

**Comment by Arcanemagus** [comment]
This is just a wrapper around `dockerlint`, do you see the same message when running that on the CLI?

**Comment by petey** [comment]
It does. I discovered that the error is due to the spaces before and after the argument inside the square brackets.

**Comment by Arcanemagus** [comment]
Good to know, thanks for finding the issue to follow!
